molgenis-data/src/main/java/org/molgenis/data/meta/model/EntityType.java

```diff
@@ -61,20 +61,6 @@ public EntityType(EntityType entityType)
 		setDefaultValues();
 	}
 
-	/**
```

Comment:
This code is actually called, I believe, when you use the EntityTypeFactory. But IntelliJ cannot detect it cause the factory uses reflection to find the constructor. Let's leave this in place.